src/dock.rs

```diff
@@ -69,10 +70,13 @@ pub async fn remove_container(docker: &Docker, id: &str) -> Result<()> {
 }
 
 pub async fn container_logs(docker: &Docker, name: &str) -> Vec<String> {
+    let tail_name = "LOGS_TAIL_VALUE";
```

Comment:
maybe renaming to `LOG_TAIL_LENGTH` might make more sense

Reply:
Ok @kevkevinpal

Reply:
@kevkevinpal I have made the change